### PR TITLE
Add sample entries

### PR DESCRIPTION
**Pull request by anubh-v**

### CODE REVIEW

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 1]
```diff
@@ -25,6 +25,51 @@
...
+                                                    .withCategory("education")
+                                                    .withTitle("National University of Singapore")
+                                                    .withDuration("2017 - 2021")
+                                                    .withSubHeader("Bachelor of Computing in Computer Science")
+                                                    .withTags("uni", "course")
+                                                    .addBulletToDescription("Attained Dean's list every year")
+                                                    .addBulletToDescription("Graduated with First Class Honours")
+                                                    .build();
+
+
```
Comment (scalarmotion) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java]: I feel like you should add some sample Minor Entries.

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 2]
```diff
@@ -122,10 +169,11 @@ public static Dictionary getSampleDictionary() {
             dictionary.registerFullPhrase("research");
             dictionary.registerFullPhrase("programme");
             dictionary.registerFullPhrase("club");
+            
 
         } catch (IllegalValueException willNotOccur) {
 
-            // the sample mappings will not throw this exception
+            assert false : "The sample mappings should not cause an IllegalValueException";
```
Comment (scalarmotion) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java]: Is this the correct syntax?

Comment (anubh-v) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java]: https://docs.oracle.com/javase/7/docs/technotes/guides/language/assert.html

Comment (anubh-v) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java]: I think it is correct right?

Comment (scalarmotion) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java]: I see.

**Pull request merged**